src/Racing.js

```diff
@@ -0,0 +1,37 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import { MIN_RANDOM_SCORE } from './constants/index.js';
+
+class Racing {
+  #totalRound = 0;
+
+  #racers = [];
+
+  constructor({ totalRound, racers }) {
+    this.#totalRound = totalRound;
+    this.#racers = racers;
```

Comment:
조금 더 깊은 복사를 위해 스프레드 연산자를 사용하시는건 어떻게 생각하시나요?
```suggestion
    this.#racers = [ ...racers ];
```